__tests__/app.test.js

```diff
@@ -0,0 +1,45 @@
+const request = require("supertest");
+const app = require("../app.js");
+const data = require("../db/data/test-data/index.js");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/ap/categories", () => {
+  describe("get request", () => {
+    test("shoul return all categories with the with the following column properties: slug and description", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.categories).toBeInstanceOf(Array)
+            expect(body.categories).toHaveLength(4)
+            expect(body.categories).toEqual([
+                { slug: 'euro game', description: 'Abstact games that involve little luck' },
+                {
+                  slug: 'social deduction',
+                  description: "Players attempt to uncover each other's hidden role"
+                },
+                { slug: 'dexterity', description: 'Games involving physical skill' },
+                { slug: "children's games", description: 'Games suitable for children' }
+              ])
```

Comment:
Assertion is too brittle here - If the sort order changes in future or more properties are added for example, this will break. Can you remember how we made assertions for the shape of objects rather than specifics?